packages/wallet/WalletManager.test.js

```diff
@@ -0,0 +1,191 @@
+const WalletManager = require('./WalletManager');
```

Comment:
We need more test cases to safeguard against wrong account being used when calling `Account` methods (in event of `this` scoping gets overlook), especially when we have both `WalletManager` and `Account` setting `this.account` in `Account.setAccount` takes in an object

Test out different scenarios involving setting accounts, accountArray(separate concern on that)
1. Standalone account instances
2. Getting different accounts from WalletManager, switching accounts for those accounts
3. Set stellar account to ethereum account, and vice-versa (Maybe in Accounts.test.js)
etc